sw/host/opentitantool/src/command/image.rs

```diff
@@ -140,6 +164,14 @@ impl CommandDispatch for ManifestVerifyCommand {
         _context: &dyn Any,
         _transport: &TransportWrapper,
     ) -> Result<Option<Box<dyn Serialize>>> {
+        let image = image::Image::read_from_file(&self.image)?;
+        let manifest: ManifestDef = image.borrow_manifest()?.try_into()?;
+        let modulus = manifest.modulus().ok_or(anyhow!("Invalid modulus"))?;
+        let signature = manifest.signature().ok_or(anyhow!("Invalid signature"))?;
```

Comment:
Thanks for adding these additional checks! Does the following work?
```suggestion
        let modulus = manifest.modulus()?;
        let signature = manifest.signature()?;
```